Sources/ETCD/ETCDClient.swift

```diff
@@ -80,9 +80,25 @@ public final class EtcdClient: @unchecked Sendable {
     
     /// Fetch the value for a key from the ETCD server.
     ///
-    /// - Parameter key: The key to fetch the value for. Parameter is of type Sequence<UInt8>.
-    /// - Returns: A `Value` containing the fetched value, or `nil` if no value was found.
+    /// - Parameter key: The key to fetch the value for. Parameter is of type String.
```

Comment:
We shouldn't delete the `Returns` documentation here

Reply:
Fixed!